### PR TITLE
README.md: Update the way `kraft up` is run

### PR DESCRIPTION
**Pull request by mogasergiu**
Change the way `kraft up` is run, as now the `-t` option seems to be mandatory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,49 @@
...
+Build the application:
+```
+$ make
+```
+
+Run the application:
+- If you built the application for `kvm`:
+```
+sudo qemu-system-x86_64 -kernel "build/app-helloworld_kvm-x86_64" \
+                        -enable-kvm
```
Comment (gabrielmocanu) [inline README.md]:
```suggestion
                        -enable-kvm \
```

**Review by gabrielmocanu — CHANGES_REQUESTED** [review]
I think it's easier to put \ there too, so that anyone can give a copy-paste.

**Review by gabrielmocanu — APPROVED** [review]
All good right now.

Reviewed-by: Gabriel Mocanu <gabi.mocanu98@gmail.com>

**Comment by razvand** [comment]
Approved-by: Razvan Deaconescu <razvan.deaconescu@cs.pub.ro>

**Pull request closed without merging**